packages/@sanity/dashboard/src/widgets/sanityTutorials/SanityTutorials.js

```diff
@@ -94,8 +94,11 @@ class SanityTutorials extends React.Component {
                     paddingX={5}
                     tone="primary"
                     as="a"
-                    href={`https://www.sanity.io/docs?ref=dashboard-${templateRepoId || 'plugin'}`}
-                    text="Go go docs"
+                    target="_blank"
+                    href={`https://www.sanity.io/docs?ref=studio-dashboard-${
+                      templateRepoId || 'plugin'
```

Comment:
Lets remove the `templateRepoId` and just hardcode it to `studio-dashboard` according to the request from @joneidejohnsen in Slack